scripts/build/VSImport-Samples.ps1

```diff
@@ -14,15 +15,18 @@ $ErrorActionPreference = "Stop"
 
 & $PSScriptRoot\unzip.ps1 -zipFile $zipFile -outputDir $tempDir
 
-$XCodeProjects = gci -recurse $tempDir\*.xcodeproj
+$XCodeProjects = gci -recurse $PSScriptRoot\..\..\samples\*.xcodeproj
+$XCodeProjects += gci -recurse $PSScriptRoot\..\..\tests\testapps\*.xcodeproj
```

Comment:
I was thinking you'd pass in \build\ and let it recurse from there

---
In reply to: [103061909](https://github.com/Microsoft/WinObjC/pull/2097#discussion_r103061909) [](ancestors = 103061909,103060288)